framework/Models/EnsembleModel.py

```diff
@@ -554,7 +554,7 @@ def _externalRun(self,inRun, jobHandler):
     for modelIn in self.orderList:
       # reset the DataObject for the projection
       self.localTargetEvaluations[modelIn].reset()
-      inRunTargetEvaluations[modelIn] = copy.copy(self.localTargetEvaluations[modelIn])
+      inRunTargetEvaluations[modelIn] = copy.deepcopy(self.localTargetEvaluations[modelIn])
```

Comment:
please put a comment here to prevent the delete of ```deepcopy```

Reply:
Is this part of the coding standards I didn't see?

Reply:
@PaulTalbot-INL it is just to avoid that somebody in the future removes it (as I did in the past). It is not a coding standard... It is just to avoid future mistakes.